externs/shaka/manifest.js

```diff
@@ -346,6 +351,11 @@ shaka.extern.CreateSegmentIndexFunction;
  *   as the value. If the channel number is not provided by the description,
  *   we'll set an 0-based index as the key.
  *   Example: {'CC1': 'eng'; 'CC3': 'swe'}, or {'1', 'eng'; '2': 'swe'}, etc.
+ * @property {(string|undefined)} tilesLayout
+ *   <i>Image streams only.</i> <br>
+ *   The value is a grid-item-dimension consisting of two positive decimal
```

Comment:
Still a bit lost about this. So, there's an image grid, and every image is at an NxM point? And they somehow correspond to - what? - timeline?

Reply:
In DASH for an audio or video segment, we have a duration, the same applies for images, with the exception that if said segment contains a tileLayout other than 1x1, it indicates that there is more than one image in that segment, so it is you have to divide the duration by the number of images and do a series of calculations to get the correct position of the image (see getThumbnails)